src/components/stepforms/schemas/filter.ts

```diff
@@ -100,7 +100,52 @@ export default {
     },
     condition: {
       type: 'object',
-      oneOf: [simpleConditionSchema, complexConditionSchema],
+      properties: {
```

Comment:
I think we should remove, for now, the "dead" code (`multipleValueConditionSchema`, etc.)

Reply:
I agree with you but I thought it was better to keep the useless code for reuse it later, well ok it's quite dirty ;)

Reply:
I thought it was better to keep the useless code for reuse it later... well ok it's quite dirty ;)